docs/index.md

```diff
@@ -2,6 +2,11 @@
 
 This code is used to implement a Standard Occupational Classification (SOC) library used by Survey Assist API.
 
+## Data sources
```

Comment:
The sic-classification-utils has a section for example data which points to the datasheets in the repo, you could do something similar for soc to keep them aligned.